test/cogctl/actions/bundles.exs

```diff
@@ -62,7 +80,8 @@ defmodule Cogctl.Actions.Bundles.Test do
 
   test "bundles info returns an error when args are missing" do
     assert run("cogctl bundles info") =~ ~r"""
-    ERROR: "Missing required arguments"
+    Usage: [\s\S]*\
+    cogctl: Missing required arguments: 'bundle'
```

Comment:
Actually no, I just forgot to include the `ERROR:` bit here. This does bring up a question I have though; do we really want to include `ERROR:` everywhere or would it be just as good just output the error message?